connectors/xai_connector.py

```
@@ -48,159 +48,157 @@ def __init__(self, api_key: Optional[str] = None):
         content = api_response["choices"][0]["message"]["content"]
-        
+
         return {
             "success": True,
             "content": content,
             "metadata": {
                 "model": api_response["model"],
                 "usage": api_response["usage"],
                 "sources_used": api_response["usage"].get("num_sources_used", 0),
-                "timestamp": datetime.now().isoformat()
+                "timestamp": datetime.now().isoformat(),
```

Ruff (0.11.9)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)